robotpy_ext/control/toggle.py

```diff
@@ -1,61 +1,62 @@
 from _functools import partial
-from wpilib.timer import Timer
+
+import wpilib
 
 
 class Toggle:
-    """Utility class for button toggle
+    """Utility class for joystick button toggle
 
-        Usage:
+        Usage::
```

Comment:
Given that the first line of the docstring is on the same line as the opening quotes, this line should also have the same indentation as the quotes.